### PR TITLE
Support Veres One type DIDs in the documentLoader.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
Need to update downstream deps:
* [ ] https://github.com/digitalbazaar/bedrock-vc-issuer
* [ ] https://github.com/digitalbazaar/bedrock-web-vc-issuer
* [ ] https://github.com/digitalbazaar/bedrock-edv-storage (probably not needed)
* [ ] https://github.com/digitalbazaar/bedrock-profile-http
* [ ] https://github.com/digitalbazaar/bedrock-web-profile-manager
* [ ] https://github.com/digitalbazaar/bedrock-profile
* [ ] https://github.com/digitalbazaar/veres-issuer

File: lib/index.js [hunk 1]
```diff
@@ -15,6 +14,13 @@ const MongoDBKeyDescriptionStorage =
   require('./MongoDBKeyDescriptionStorage.js');
 require('bedrock-did-context');
 
+const didIo = require('did-io');
+
+// Config did-io to support did:key and did:v1 drivers
+didIo.use('key', require('did-method-key').driver());
+// TODO: Move v1 'test' mode param to config.
+didIo.use('v1', require('did-veres-one').driver({mode: 'test'}));
```
Comment (mattcollier) [inline lib/index.js:22]: How do things go when we want to use some other V1 mode?

Comment (dmitrizagidulin) [reply lib/index.js:22]: @mattcollier good point. I think the mode is there just to be the default when generating new DIDs. For resolving (which is what this is doing), we shouldn't need to specify, driver should route based on the DID itself (and the presence of `test` in it).
Double-checking..

Comment (dlongley) [reply lib/index.js:22]: Yeah, configuring a single implicit instance of "didIo" like this is problematic across multiple modules. I approved for the near term, but we will need a number of architectural fixes in the future.

**Comment by mattcollier** [comment]
@dmitrizagidulin what is the status of this PR?

File: lib/index.js (hunk 1 above)
Comment (dlongley) [reply lib/index.js:23]: @mattcollier -- Do we want to merge with this and open an issue to address it later?

File: lib/index.js [hunk 2]
```diff
@@ -42,8 +49,13 @@ bedrock.events.on('bedrock.init', async () => {
 const defaultModuleManager = new BedrockKmsModuleManager();
 const defaultStorage = new MongoDBKeyDescriptionStorage();
 const defaultDocumentLoader = async url => {
-  if(url.startsWith('did:key:')) {
-    const document = await didKeyDriver.get({url});
+  let document;
+  if(url.startsWith('did:')) {
+    document = await didIo.get({did: url, forceConstruct: true});
```
Comment (dlongley) [inline lib/index.js:54]: If we merge, this will also need an issue.

**Pull request merged**